### ISSUE TITLE
Gem does not install properly

### ISSUE THREAD

**Issue by sesshomariu** [packetfu/packetfu#168]
I installed the gem with `sudo gem install packetfu --no-user-install`, there were no error messages. Then I tried to start the REPL, also as root. But it didn't let me. The error message was as follows:
```
/usr/lib/ruby/gems/2.4.0/gems/packetfu-1.1.12/lib/packetfu/packet.rb:47:in `parse': uninitialized constant PacketFu::IPv6Packet (NameError)
Did you mean?  PacketFu::Packet
        from /usr/lib/ruby/gems/2.4.0/gems/packetfu-1.1.12/lib/packetfu/utils.rb:148:in `block in whoami?'
        from /usr/lib/ruby/2.4.0/timeout.rb:93:in `block in timeout'
        from /usr/lib/ruby/2.4.0/timeout.rb:33:in `block in catch'
        from /usr/lib/ruby/2.4.0/timeout.rb:33:in `catch'
        from /usr/lib/ruby/2.4.0/timeout.rb:33:in `catch'
        from /usr/lib/ruby/2.4.0/timeout.rb:108:in `timeout'
        from /usr/lib/ruby/gems/2.4.0/gems/packetfu-1.1.12/lib/packetfu/utils.rb:141:in `whoami?'
        from packetfu-shell.rb:70:in `<main>'
```
What did I miss?

I use Arch Linux.

(I know, this really seems like a childish issue, but I install a lot of gems and it always works for me, so this really isn't normal)

**Comment by claudijd** [comment]
I tried testing this on current master (2e6652b39ef32052713f0725ef517f58f66c0cb0) and I'm not seeing a loading issue when trying to loading packetfu-shell.  I'll test the gem next and see what I figure out for you.

```ruby
$ rvmsudo ruby -I lib ./examples/packetfu-shell.rb
 _______  _______  _______  _        _______ _________ _______
(  ____ )(  ___  )(  ____ \| \    /\(  ____ \\__   __/(  ____ \|\     /|
| (    )|| (   ) || (    \/|  \  / /| (    \/   ) (   | (    \/| )   ( |
| (____)|| (___) || |      |  (_/ / | (__       | |   | (__    | |   | |
|  _____)|  ___  || |      |   _ (  |  __)      | |   |  __)   | |   | |
| (      | (   ) || |      |  ( \ \ | (         | |   | (      | |   | |
| )      | )   ( || (____/\|  /  \ \| (____/\   | |   | )      | (___) |
|/       |/     \|(_______/|_/    \/(_______/   )_(   |/       (_______)
 ____________________________              ____________________________
(                            )            (                            )
| 01000001 00101101 01001000 )( )( )( )( )( 00101101 01000001 00100001 |
|                            )( )( )( )( )(                            |
(____________________________)            (____________________________)
                               PacketFu
             a mid-level packet manipulation library for ruby

>>> PacketFu Shell 1.1.13.pre.
>>> Use $packetfu_default.config for salient networking details.
IP:  192.168.10.37   Mac: ac:bc:32:85:47:3f   Gateway: 00:60:e0:51:f0:08
Net: 192.168.10.0                             Iface:   en0
>>> Packet capturing/injecting enabled.
<><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
2.2.3 :001 > ```

**Comment by claudijd** [comment]
I was able to test the lib loading successfully after installing the 1.1.13.pre gem...

```bash
$ gem list | grep packetfu
packetfu (1.1.13.pre)
$ rvmsudo irb
2.2.3 :001 > require 'packetfu'
 => true

**Comment by claudijd** [comment]
I was also able to test the lib loading successfully after installing the 1.1.12 gem...

```bash
$ gem list | grep packetfu
packetfu (1.1.12)
$ rvmsudo irb
2.2.3 :001 > require 'packetfu'
 => true

**Comment by claudijd** [comment]
err... those tests were on 2.2.3, switching to 2.4.0 to see if that behaves the same as above.

**Comment by claudijd** [comment]
Confirmed that Ruby 2.4.0 loads PacketFu just fine on 1.1.12

```bash
$ ruby -v
ruby 2.4.0p0 (2016-12-24 revision 57164) [x86_64-darwin14]
$ rvmsudo irb
Password:
2.4.0 :001 > require 'packetfu'
 => true

**Comment by claudijd** [comment]
@sesshomariu can you please provide repo steps for the above trace you sent?  I'm having trouble replicating your issue.  Hopefully with some more details we can get to the bottom of this.

**Comment by claudijd** [comment]
I believe the label on this issue is sort of misleading, but in staring at the trace and hypothesizing about how and why this might happen I've come to the conclusion that this would be fixed by #144, which I neglected to land for a very long time.  Assuming no issues, we can probably put out new gem soon to address this.

**Comment by claudijd** [comment]
Closing under the pretense that #144 fixes this which has been merged into master.
